src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/RecommendationComponents/CostDetails.tsx

```diff
@@ -87,9 +88,47 @@ const CostDetails = (props: { recommendedData; currentData; chartData; day; endt
       NumberFormat(props.recommendedData[0]?.recommendation_engines?.cost?.variation?.limits?.cpu?.amount)
     )}`;
 
+  /// Alert for over and under utlized
+  useEffect(() => {
+    if (props.recommendedData !== null) {
+      utilizationAlert(props.recommendedData);
+    }
+  }, [props.recommendedData]);
+
+  type alertVarient =  'success' | 'danger' | 'warning' | 'info' | 'custom';
+ 
+  const [showSuccessAlert, setShowSuccessAlert] = useState(false);
+  const [alertMessage, setAlertMessage] = useState('');
+  const [notificationType, setNotificationType] = useState<alertVarient>('info');
+
+  const utilizationAlert = (recommendation) => {
+    const notifications = recommendation[0]?.recommendation_engines?.cost?.notifications;
+    try {
+      Object.values(notifications).forEach((notification : any, index) => {
+        setTimeout(() => {
+          setAlertMessage(`${notification.code} - ${notification.message}`);
+          if(notification.type == "notice") {
+            setNotificationType('info'); 
+          }
+          else if (notification.type == "error"){
+            setNotificationType('danger');
```

Comment:
Any reason on setting a different value for notification type ? 
If notification.type is "error" , why to set as "danger" instead of "error" itself ?

Reply:
patternfly has "info" and "Danger" as the notification variants while from the backend we get "notice" and "error", so over here im just trying to match the notification type from backend to the patternfly notifications. reference https://www.patternfly.org/components/alert

Reply:
As I don't see warning and critical related notifications, can you please add them as well.

Reply:
Added